edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/TwinManager.cs

```diff
@@ -82,6 +82,26 @@ public async Task<IMessage> GetTwinAsync(string id)
                 });
         }
 
+        public async Task<Option<IMessage>> GetLocalTwinAsync(string id)
+        {
+            return await this.TwinStore.Match(
```

Comment:
If you instead use Map, it will automatically return an Option, no?

Reply:
the issue I had with Map is when twin is null I could not return Option.None

Reply:
I would return a Option<Task<Option<IMessage>>>